src/System.Collections.Concurrent/tests/ConcurrentDictionary/ConcurrentDictionaryTests.cs

```diff
@@ -546,25 +546,51 @@ public static void TestDebuggerAttributes()
         }
 
         [Fact]
-        public static void TestConstructor_Negative()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework hasn't received the fix for https://github.com/dotnet/corefx/issues/18432 yet.")]
+        public static void TestNullComparer()
```

Comment:
This test would ideally do some verification that the default comparer is being used in place of the supplied null.